tests/filecheck/dialects/affine/affine_ops.mlir

```diff
@@ -30,4 +30,19 @@
     // CHECK-NEXT:   "affine.yield"(%{{.*}}) : (!test.type<"int">) -> ()
     // CHECK-NEXT: }) {"lower_bound" = -10 : index, "upper_bound" = 10 : index, "step" = 1 : index} : (!test.type<"int">) -> !test.type<"int">
 
+    %memref = "test.op"() : () -> memref<2x3xf64>
+    %value = "test.op"() : () -> f64
+    "affine.store"(%value, %memref) {"map" = affine_map<() -> (0, 0)>} : (f64, memref<2x3xf64>) -> ()
```

Comment:
Isn't the map wrong? Shouldn't it be affine_map<(d0, d1) -> (0, 0)>?

Reply:
When I change it it throws an error. But also it doesn't surprise me, what would those values be?